upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -801,6 +801,21 @@ There you can find links to upgrade notes for other versions too.
         +     }
         + },
         ```
+    - contact form has been moved to separate page. You can find the whole new setting in administration (`/admin/contact-form/`), where you can edit main text for contact form.
+    There are few steps need to be done, to make contact form work on FE
+        - in `ContactFormController` change previous `indexAction` with [the new one]((https://github.com/shopsys/shopsys/blob/master/project-base/src/Shopsys/ShopBundle/Controller/Front/ContactFormController.php)). Please pay attention if you have some modification in previous implementation.
+        - in `ContactFormController` remove action `sendAction()`, it is not needed anymore
+        - remove `/src/Shopsys/ShopBundle/Resources/scripts/frontend/contactForm.js`, it is not needed anymore
+        - add new localized route in all your localized routing `yml` files with translated `path` option
+            ```diff 
+            +    front_contact:
+            +    path: /contact/
+            +    defaults: { _controller: ShopsysShopBundle:Front\ContactForm:index }
+            ```
+        - add new [twig template](https://github.com/shopsys/shopsys/blob/master/project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/ContactForm/index.html.twig)
+        - add link to new contact page somewhere in templates (e.g in `footer.html.twig`)
+        - it is recommended to create database migration to remove `contact` slug from `friendly urls` database table, to make new url work. 
```

Comment:
the migration exists, why not link it? 
it's recommended or it's mandatory? it's recommended means to me I don't need to do it, but here it will not work at all, or?

Reply:
link has been added with info to move content from previous page to the new contact page